core/templates/dev/head/galleries/Gallery.js

```diff
@@ -95,16 +95,32 @@ oppia.controller('Gallery', [
     });
 
     $scope.allExplorationsInOrder = [];
+    $scope.allCollectionsInOrder = [];
```

Comment:
Yeah, we should have one list. I'm not sure whether an activity-summary-tile is the right way to go, though. I think think it is, since in this case the two different directives are used for linking to different types of activities, plus the summary cards are displayed differently for each. I think long term we will have frontend domain objects for ``ExplorationSummary`` and ``CollectionSummary`` which may even derive from a common ``ActivitySummary`` that could address this issue. For now, I think it would be okay just to differentiate activity type within this list; thoughts? If you would like to discuss this further before implementing, feel free to ping me.